Bottom: f059a10b91ef4f8b88e55360bedc02bf7ca4f601
Top:    f376f001d88c9c603a1530c5b84c467bc13c758f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-06 16:14:30 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
